<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurveyItemLoader</class>
 <widget class="QDialog" name="SurveyItemLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>distanceToSurfaceRelative:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>imageHeight:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>distanceToSurface:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="imageWidthSpinBox">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>FootprintFrontal:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>imageWidth:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDoubleSpinBox" name="sensorWidthDoubleSpinBox"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>frontalOverlap:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>sensorHeight:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="distanceToSurfaceSpinBox">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Camera Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="distanceToSurfaceRelativeComboBox">
     <item>
      <property name="text">
       <string>false</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>true</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDoubleSpinBox" name="focalLengthDoubleSpinBox"/>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>imageDensity:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="landscapeComboBox">
     <item>
      <property name="text">
       <string>false</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>true</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="footprintFrontaDoubleSpinBox"/>
   </item>
   <item row="9" column="3">
    <widget class="QComboBox" name="valueSetIsDistanceComboBox">
     <item>
      <property name="text">
       <string>false</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>true</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>FootprintSide:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="fixedOrientationComboBox">
     <item>
      <property name="text">
       <string>false</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>true</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>landscape:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QSpinBox" name="imageHeightSpinBox">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>sequence:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QDoubleSpinBox" name="imageDensityDoubleSpinBox">
     <property name="maximum">
      <double>65535.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QDoubleSpinBox" name="sensorHeightDoubleSpinBox"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>items:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSpinBox" name="itemsSpinBox">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="minTriggerInteralSpinBox">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="frontIOverlapSpinBox"/>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>valueSetIsDistance:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>minTriggerInteral:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>fixedOrientationï¼š</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>focalLength:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>sideOverLap:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="footprintSideDoubleSpinBox"/>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="cameraNameLineEdit"/>
   </item>
   <item row="11" column="3">
    <widget class="QSpinBox" name="sideOverlapSpinBox"/>
   </item>
   <item row="15" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>sensorWidth:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sequenceSpinBox">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>params:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="4">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>altitude:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>longitude:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>latitude:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="latitudeDoubleSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="longitudeDoubleSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>speed:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="speedDoubleSpinBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>radius:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="radiusSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="flagSpinBox">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>flag</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="altitudeSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SurveyItemLoader</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SurveyItemLoader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
